requirements/dev.txt

```diff
@@ -4,83 +4,70 @@
 #
 #    pip-compile --output-file requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in requirements/test.in requirements/travis.in
 #
--e git+https://github.com/edx/django-oauth2-provider.git@0.2.7-fork-edx-6a#egg=django-oauth2-provider==0.2.7-fork-edx-6
 -e git+https://github.com/edx/django-rest-framework-oauth.git@f0b503fda8c254a38f97fef802ded4f5fe367f7a#egg=djangorestframework-oauth
 -e git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
 appdirs==1.4.3            # via fs
 argparse==1.4.0           # via caniusepython3
-asn1crypto==0.24.0        # via cryptography
 astroid==1.5.3            # via edx-lint, pylint, pylint-celery
-backports.functools-lru-cache==1.5  # via astroid, caniusepython3, pylint
```

Comment:
I'm guessing you ran `make upgrade` under Python 3?  Given that none of the packages depending on this were upgraded, I think this is still needed under Python 2.7.  We'll probably need to keep running `make upgrade` under 2.7 until either we drop support for it or `pip-tools` gets smarter about noticing different requirements under different Python versions.  I suspect several of the other dependencies that were removed from these files are in a similar situation.

Reply:
You guessed right and I did not realize that makes a difference. Thanks a lot for catching this!